pkg/reconciler/kubernetes/tektonconfig/extension.go

```diff
@@ -52,13 +51,7 @@ func (oe kubernetesExtension) PostReconcile(ctx context.Context, comp v1alpha1.T
 	}
 
 	if configInstance.Spec.Profile == common.ProfileLite || configInstance.Spec.Profile == common.ProfileBasic {
-		err := extension.TektonDashboardCRDelete(oe.operatorClientSet.OperatorV1alpha1().TektonDashboards(), common.DashboardResourceName)
-		if err != nil {
-			if errors.IsNotFound(err) {
-				return nil
-			}
-			return err
-		}
+		return extension.TektonDashboardCRDelete(oe.operatorClientSet.OperatorV1alpha1().TektonDashboards(), common.DashboardResourceName)
```

Comment:
i think, here the code should ignore notFound error.

if initially the profile was `LITE` and then it was changed to `BASIC`, then we will get a `NotFound` error for delete Dashboard, as Dashboard is not provided in either of the profiles.

Reply:
actually the not found case is already handled inside `TektonDashboardCRDelete` so we don't need to explicitly handle it :)

Reply:
oh that is wonderful 🧑‍💻 👍